recipes/client.rb

```diff
@@ -173,13 +173,15 @@
 zap_directory '/etc/sensu/conf.d/checks' do
   pattern '*.json'
   notifies :create, 'ruby_block[sensu_service_trigger]', :immediately
+  not_if { node[:platform_family].include?("windows") }
 end
 
 include_recipe 'sensu::client_service' unless node['recipes'].include?('monitor::master')
 
 directory '/var/cache/sensu' do
   owner 'sensu'
   group 'sensu'
-  mode 0o0755
+  mode 0755
```

Comment:
I thought it was a Linux <-> Windows format conversion issue. If that is not the case, there is no need to change it.